web/modules/custom/nidirect_common/nidirect_common.install

```diff
@@ -322,6 +322,25 @@ function nidirect_common_update_8005() {
 }
 
 
+/**
+ * Copy telephone field values to phone field to fix issue raise in D8NID-1678.
+ */
+function nidirect_common_update_8006() {
+  $database = \Drupal::database();
+
+  // Use 'IGNORE' to prevent 'Duplicate entry' warnings.
```

Comment:
Nice, this looks a good bit simpler than using some kind of intermediary field as storage